tests/topotests/bgp_snmp_mplsl3vpn/test_bgp_snmp_mplsvpn.py

```diff
@@ -161,13 +161,6 @@ def setup_module(mod):
     r1.run("sysctl -w net.mpls.conf.r1-eth0.input=1")
     r1.run("sysctl -w net.mpls.conf.r1-eth1.input=1")
     r1.run("sysctl -w net.mpls.conf.r1-eth2.input=1")
-    r2.run("sysctl -w net.mpls.conf.r1-eth0.input=1")
-    r2.run("sysctl -w net.mpls.conf.r1-eth1.input=1")
-    r3.run("sysctl -w net.mpls.conf.r1-eth0.input=1")
-    r3.run("sysctl -w net.mpls.conf.r1-eth1.input=1")
-    r3.run("sysctl -w net.mpls.conf.r1-eth2.input=1")
-    r4.run("sysctl -w net.mpls.conf.r1-eth0.input=1")
-    r4.run("sysctl -w net.mpls.conf.r1-eth1.input=1")
```

Comment:
I think instead of deletion we should set correct interface names?
```
    r2.run("sysctl -w net.mpls.conf.r2-eth0.input=1")
    r2.run("sysctl -w net.mpls.conf.r2-eth1.input=1")
    r3.run("sysctl -w net.mpls.conf.r3-eth0.input=1")
    r3.run("sysctl -w net.mpls.conf.r3-eth1.input=1")
    r3.run("sysctl -w net.mpls.conf.r3-eth2.input=1")
    r4.run("sysctl -w net.mpls.conf.r4-eth0.input=1")
    r4.run("sysctl -w net.mpls.conf.r4-eth1.input=1")
```

Reply:
if the test is already working *without* setting mpls up on these interfaces then we *know* that this code is not needed to be run.  Hence the removal